### ISSUE TITLE
GN 2.4.1 métadonnées: affichage aléatoire du diagramme de répartition des espèces d'un JDD

### ISSUE THREAD

**Issue by pguillermin** [PnX-SI/GeoNature#959]
Bonjour,

J'ai constaté un comportement aléatoire de l'affichage du diagramme de répartition des espèces lorsque l'on consulte les métadonnées avec geonature 2.4.1.

Certains JDD bien fournis avec plusieurs taxons n'affichent pas les catégories d'espèces dans le diagramme qui apparaît en gris. Il y a un chiffre de récupéré ex undefined: 3.

![2020-06-26_15-49](https://user-images.githubusercontent.com/31274064/85864900-6807bf80-b7c5-11ea-8f8b-a81ac5b2ec52.png)

Lorsque je rafraîchit plusieurs fois la page, ou si j'accède directement à la fiche par l'url, les couleurs et le nom des catégories d'espèces apparaissent.

![2020-06-26_15-51](https://user-images.githubusercontent.com/31274064/85864931-7524ae80-b7c5-11ea-905e-8d19e6521d2b.png)

Apparemment il n'y a pas de message d'erreur dans les logs.

**Comment by camillemonchicourt** [comment]
Oui j'ai aussi identifié que le graphique avait un comportement aléatoire, il doit pas arriver à charger les données à temps. Le problème est côté du navigateur, donc pas dans les logs, éventuellement dans la console de ton navigateur web. A creuser.
Super, si tu as déjà pu passer sur la version 2.4.1 !

**Comment by pguillermin** [comment]
Oui, mise à jour réalisée hier par @Splendens, encore merci à elle.
